content/grants/2023/community-meetups.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "Community Meetup"
+date = "2023-04-06"
+
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "Community" ]
+
+[extra]
+image = ""
+description = "Earn stars by building a local urbit community"
+reward = "1 star"
+assignee = [""]
+grant_id = "B0165"
+champion = ["~dalweb-donfun"]
+completed = false
+work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0165&prefill_Grant+Name=Meetup%20Bounty"
++++
+
+# Requirements
+- Meetups should be geographically based.
+- You must host a total of six meetups, with a targeted cadence of once a month.
+- Meetups must be publicized and include a web presence like meetup.com, and should be actively promoted via social media, in-person recruitment, flyers, a website, etc. If you need a signal boost, email thomas@urbit.org.
+- Create and admin an Urbit group for your community to which all meetup attendees will be invited. This group should be actively maintained with info about upcoming events, copies of past presentation materials, and any other related meetup content.
+- Share all the materials you generate in service of your meetups like, presentations, flyers, FAQs or anything else in ~siddef/uf-meetups.
+- Every meetup must be publicized on the [Events](https://urbit.org/events) section on urbit.org via a commit to the urbit.org [github repo](https://github.com/urbit/urbit.org). 
+- Your community should have a home on the [community section](https://urbit.org/ecosystem?type=communities) of our ecosystem page. You can add this to the website with a commit to the urbit.org [github repo](https://github.com/urbit/urbit.org). If you have never submitted a pull request on github before, let us know and we will teach you how.
+- Write a short meetup report about all meetups including: pictures of the meetups, 50-200 words description of every meetup and the attendees statistics of the meetups. At the end of your grant you can send this to thomas@urbit.org for completion of your grant.  
```

Comment:
Maybe "write a short report after each meetup"?

Reply:
Yes, I just don't want them to hand it in for every meetup, that increases our burden

Reply:
Oh I missed the intent. Sounds good then.